redash/query_runner/activedata.py

```diff
@@ -0,0 +1,180 @@
+import json
+import logging
+from urlparse import urlparse
+
+import requests
+# from mo_dots import wrap, listwrap
+# from mo_logs import Except
+# from mo_logs.exceptions import ERROR
+
+from redash.query_runner import TYPE_INTEGER, TYPE_STRING, TYPE_FLOAT, BaseSQLQueryRunner, register
+from redash.utils import JSONEncoder
+
+#originally from: https://github.com/klahnakoski/ActiveData-redash-query-runner/blob/c0e7286c09c6f1eb6746a6c7cca581bea79f4757/active_data.py
+
+logger = logging.getLogger(__name__)
+
+if not unicode:
+    unicode = str
+
+types_map = {
+    bool: TYPE_INTEGER,
+    str: TYPE_STRING,
+    unicode: TYPE_STRING,
+    dict: TYPE_STRING,
+    list: TYPE_STRING,
+    int: TYPE_INTEGER,
+    long: TYPE_INTEGER,
+    float: TYPE_FLOAT
+}
+
+
+class ActiveData(BaseSQLQueryRunner):
+    noop_query = "SELECT 1"
+
+    def __init__(self, configuration):
+        super(ActiveData, self).__init__(configuration)
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "host_url": {
+                    "type": "string",
+                    "title": "Host URL",
+                    "default": "https://activedata.allizom.org:80",
+                    "info": "Do not end with a trailing slash."
+                },
+                "doc_url": {
+                    "type": "string",
+                    "title": "Documentation URL",
+                    "default": "https://github.com/klahnakoski/ActiveData/tree/dev/docs"
+                }
+            },
+            "required": ["host_url"]
+        }
+
+    @classmethod
+    def name(cls):
+        return "ActiveData"
+
+    @classmethod
+    def type(cls):
+        return "activedata"
+
+    @classmethod
+    def enabled(cls):
+        return True
+
+    def _get_tables(self, schema):
+        query = {
+            "from": "meta.columns",
+            "select": [
+                "name",
+                "table"
+            ],
+            "where": {"not": {"prefix": {"es_index": "meta."}}},
+            "limit": 1000,
```

Comment:
Probably not a big deal and can leave as-is if you think that's a good default. I was just imagining if this were an installable Python package and redash has ways to pass configuration parameters, this seemed like a good candidate.